pkg/utils/cluster_test.go

```diff
@@ -78,10 +78,10 @@ hello: world
 func writeKubeconfigYaml(s string) error {
 	kubeconfigPath := clientcmd.NewDefaultPathOptions().GlobalFile
 	dirname := filepath.Dir(kubeconfigPath)
-	err := os.MkdirAll(dirname, 0744)
-	if err != nil {
-		return err
-	}
+	if _, err := os.Stat(dirname); os.IsNotExist(err) {
+		os.MkdirAll(dirname, 0777)
+		os.Chmod(dirname, 0777)
```

Comment:
You will have to handle error properly here, i.e to return it 
```suggestion
		os.Chmod(dirname, 0777)
		if err!=nil{
			return err
		}
```